docs/en/siem/installation.asciidoc

```diff
@@ -30,13 +31,77 @@ for sorting. For more information, see {kib}.
 [[install-beats]]
 === Install {beats} shippers
 
-To populate the security UI with metrics and
-log data, you need to install and configure the following shippers:
+To populate the {siem-ui} with hosts and network data, you need to install and
+configure Beats on the endpoints you want to collect and send data:
 
-* https://www.elastic.co/products/beats/packetbeat[{packetbeat}] for analyzing
-network packets 
 * https://www.elastic.co/products/beats/filebeat[{filebeat}] for forwarding and
 centralizing logs and files
 * https://www.elastic.co/products/beats/auditbeat[{auditbeat}] for monitoring
```

Comment:
Auditbeat is more than that. Perhaps, `Auditbeat for collecting security events`.